### DISCUSSION TITLE
stop drag event on large buttons

### DISCUSSION THREAD

**Discussion by LadyPsy** [RomanKubiak/ctrlr#473] [Q&A]
Hi everyone,
I am using a multi touch screen and a large mpc style button the drag event prevents the click event in the heat of performance. 
Can the drag event be stopped on this component?
Thank you

**Comment by Ted-juh** [comment]
Afaik drag events are enabled from the get go in case you want to make a component available for drag and drop. No way to disable that. To make matters worse most multi touch gestures are controlled by the OS and might override the ones in Juce.

What version of CTRLR are you on and which platform? There have been numerous fixes for multitouch input in Juce but those might not be available in CTRLR 5.3.201.. perhaps the newer 6 version of CTRLR might contain those fixes.

Or could you upload the panel in a zip file?

**Reply by LadyPsy** [reply]
Thank you for your response.
Version = 5.3.201, Build date = Wed Mar 30 00:52:44 CEDT 2016, Branch = Nightly, Juce = 4.0.2, libusb = 1.0.19, liblo = 0.28, 

the button is an image button which is about 2 fingers wide.
the screen i test on is a iiyama 24 inch.

I will give the version 6 a try. Thank you again for your help.
